src/shared/varlink-io.systemd.MountFileSystem.c

```diff
@@ -42,7 +42,9 @@ static SD_VARLINK_DEFINE_STRUCT_TYPE(
                 SD_VARLINK_FIELD_COMMENT("The start offset of the partition in bytes."),
                 SD_VARLINK_DEFINE_FIELD(offset, SD_VARLINK_INT, 0),
                 SD_VARLINK_FIELD_COMMENT("A mount file descriptor to assign to a location."),
-                SD_VARLINK_DEFINE_FIELD(mountFileDescriptor, SD_VARLINK_INT, 0));
+                SD_VARLINK_DEFINE_FIELD(mountFileDescriptor, SD_VARLINK_INT, 0),
+                SD_VARLINK_FIELD_COMMENT("A relative path indicating the intended mount point for this file system, if applicable. May contain multiple paths, for certain partitions that can be mounted to two distinct places."),
```

Comment:
"mounted to multiple distinct places" (or if it's really just two, "May contain two paths")